### PR TITLE
N8N-2942 new format google ads

### PR DESCRIPTION
**Pull request by agobrech**

### CODE REVIEW

File: packages/nodes-base/nodes/Google/Ads/GoogleAds.node.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+						name: 'Campaign',
+						value: 'campaign',
+					},
+					{
+						name: 'Invoice',
+						value: 'invoice',
+					},
+					{
+						name: 'Analytics',
+						value: 'analytic',
```
Comment (krynble) [inline packages/nodes-base/nodes/Google/Ads/GoogleAds.node.ts:69]: Keep this name consistent with the label to make it easier, so use `analytics` everywhere.

File: packages/nodes-base/nodes/Google/Ads/InvoiceDescription.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+		displayOptions: {
+			show: {
+				resource: [
+					'invoice',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Return invoices',
```
Comment (krynble) [inline packages/nodes-base/nodes/Google/Ads/InvoiceDescription.ts:19]: We can name this one "Search"

**Review by krynble — CHANGES_REQUESTED** [review]

File: packages/nodes-base/nodes/Google/Ads/GoogleAds.node.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+		inputs: ['main'],
+		outputs: ['main'],
+		credentials: [
+			{
+				name: 'googleAdsOAuth2Api',
+				required: true,
+				testedBy: {
+					request: {
+						method: 'GET',
+						url: '/v9/customers:listAccessibleCustomers',
```
[the remark this answers is not in the document]
Comment (agobrech) [reply packages/nodes-base/nodes/Google/Ads/GoogleAds.node.ts:41]: Yes it does work on my end

File: packages/nodes-base/nodes/Google/Ads/InvoiceDescription.ts [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+			},
+		},
+		default: '',
+		description: 'The year of the invoices to return in YYYY format. Only invoices issued in 2019 or later can be retrieved.',
+	},
+	{
+		displayName: 'Issue Month',
+		name: 'issueMonth',
+		type: 'string',
+		required: true,
```
[the remark this answers is not in the document]
Comment (agobrech) [reply packages/nodes-base/nodes/Google/Ads/InvoiceDescription.ts:96]: Yes all the fields are required

**Pull request closed without merging**